common/changes/fixPopupCloseButtonRootProps_2017-01-11-16-17.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Clean deprecated rootProps from Dialog Close button.",
```

Comment:
Please put in the form "Dialog: Removed deprecated rootProps from the close button."